packages/tools/release-utils-internal/src/git.ts

```diff
@@ -39,3 +42,37 @@ export async function addRemoteWithAuthentication(repoUrl: string, remoteName: s
 
     await execShellCommand(`git remote add "${remoteName}" "${getGHRepoAuthUrl(repoUrl)}"`);
 }
+
+export function noChanges(): boolean {
+    return /nothing to commit/i.test(sh.exec("git commit --dry-run -uno", { silent: true }));
+}
+
+export function commit(msg: string): void {
+    if (commitMessageRegEx.test(msg)) {
+        sh.exec(`git commit -m ${msg}`);
```

Comment:
Why not `execShellCommand` here?

Reply:
If we want to go with `sh.exec`, we should convert other places to use it also. It needs consistency.